dependency-graph/dependency-review.yml

```diff
@@ -0,0 +1,11 @@
+name: 'Dependency Review'
+on: [pull_request]
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@main
```

Comment:
TODO: before GA next week, when we plan to merge this PR, it looks from the PR template docs that we'll need to replace `main` tag with a fixed commit SHA?
1

Can you confirm this is the case? If so once the `actions/dependency-review-action` repo is released we can update to a fixed SHA/tagged release as needed 👍

Reply:
Commit SHA is not needed for first party actions. You can use a version tag though instead of `main`

Reply:
cc @febuiles @hmaurer @lseppala as per open-source guidelines, are we obligated to semver-tag changes to the DR Action so end users can track fixed versions, or can we rely on `main` here to propagate bugfixes/DR Action changes to all installs?

🤔 The former feels like the right play, but the latter will help us avoid a long-tail support burden of folks who aren't upgrading regularly

Reply:
@elireisman as discussed, since the ref used by an Action isn't required to be immutable yet (that is, you can use a branch rather than a tag), I think we should accommodate semver _and_ also have a ref that can be updated with bugfixes. To keep it simple, we can call it `v1`

@febuiles, this will require a small amount of work in the Action repo. We'll need a _branch_ named `v1` to match `main`, which is what this Action will track.

Reply:
@lseppala @febuiles c.f. https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management